mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://json.schemastore.org/mkdocs-1.6.json
 site_name: PrairieLearn Docs
-site_url: https://prairielearn.readthedocs.io
+site_url: https://prairielearn.readthedocs.io/en/latest/
```

Comment:
This is correct, I'm hoping this fixes the 404 of assets. If not, it is still strictly more correct.